src/main/resources/db/changelog/changes/v0001.sql

```diff
@@ -0,0 +1,38 @@
+CREATE SEQUENCE logging_event_id_seq MINVALUE 1 START 1;
```

Comment:
is this file autogenerated? if so, remove it from git tracking. 
If not, why do we even have it? Logger creates DB itself, isn't it?

Reply:
no, it isn't. in case of logback we should to create necessary tables. see https://github.com/qos-ch/logback/tree/master/logback-classic/src/main/resources/ch/qos/logback/classic/db/script